samples/peripheral/802154_phy_test/src/ctrl/ptt_events.c

```diff
@@ -0,0 +1,307 @@
+/* Copyright (c) 2019, Nordic Semiconductor ASA
+ * All rights reserved.
+ *
+ * Use in source and binary forms, redistribution in binary form only, with
+ * or without modification, are permitted provided that the following conditions
+ * are met:
+ *
+ * 1. Redistributions in binary form, except as embedded into a Nordic
+ *    Semiconductor ASA integrated circuit in a product or a software update for
+ *    such product, must reproduce the above copyright notice, this list of
+ *    conditions and the following disclaimer in the documentation and/or other
+ *    materials provided with the distribution.
+ *
+ * 2. Neither the name of Nordic Semiconductor ASA nor the names of its
+ *    contributors may be used to endorse or promote products derived from this
+ *    software without specific prior written permission.
+ *
+ * 3. This software, with or without modification, must only be used with a Nordic
+ *    Semiconductor ASA integrated circuit.
+ *
+ * 4. Any software provided in binary form under this license must not be reverse
+ *    engineered, decompiled, modified and/or disassembled.
+ *
+ * THIS SOFTWARE IS PROVIDED BY NORDIC SEMICONDUCTOR ASA "AS IS" AND ANY EXPRESS OR
+ * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+ * MERCHANTABILITY, NONINFRINGEMENT, AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL NORDIC SEMICONDUCTOR ASA OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+/* Purpose: events implementation */
+
+#include <assert.h>
+#include <stddef.h>
+#include <string.h>
+
+#include "ctrl/ptt_trace.h"
+
+#include "ptt_events_internal.h"
+
+#ifdef TESTS
+#include "test_events_conf.h"
+#else
+#include "ptt_ctrl_internal.h"
+#endif
+
+void ptt_events_init(void)
+{
+	PTT_TRACE("%s ->\n", __func__);
+
+	struct ptt_evt_ctx_s *p_evt_ctx = ptt_ctrl_get_p_evt_ctx();
+
+	*p_evt_ctx = (struct ptt_evt_ctx_s){0};
+
+	PTT_TRACE("%s -<\n", __func__);
+}
+
+void ptt_events_uninit(void)
+{
+	PTT_TRACE("%s\n", __func__);
+}
+
+void ptt_events_reset_all(void)
+{
+	PTT_TRACE("%s ->\n", __func__);
+
+	ptt_events_uninit();
+	ptt_events_init();
+
+	PTT_TRACE("%s -<\n", __func__);
+}
+
+enum ptt_ret_t ptt_event_alloc(ptt_evt_id_t *p_evt_id)
+{
+	PTT_TRACE("%s -> p_evt %p\n", __func__, p_evt_id);
+
+	uint8_t         i;
+	struct ptt_evt_ctx_s *p_evt_ctx = ptt_ctrl_get_p_evt_ctx();
+	enum ptt_ret_t       ret       = PTT_RET_SUCCESS;
```

Comment:
This whole alignment seems totally wrong. Anyway, you don't need to align variable declarations/asignments in NCS so you can just skip it.

Reply:
Removed alignment